docker/package/scripts/tezos-baking-custom-reset

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# SPDX-FileCopyrightText: 2022 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+set -euo pipefail
+
+if [ -z "$1" ]; then
+    echo "Please call this script with the custom baking instance name."
+    exit 1
+fi
+
+instance_name="$1"
+systemctl stop tezos-baking-custom@"$instance_name"
+node_data_dir="$(systemctl show tezos-node-custom@"$instance_name".service | grep -o "NODE_DATA_DIR=\S*" | cut -d'=' -f2)"
+rm -rf "$node_data_dir"
```

Comment:
This is the only choice that seems a bit odd to me.
AFAIK the purpose of this service is mainly this step: to clean up the node data dir.

I'm not familiar with oneshot units, and it seems that cleanup is a common use-case for them, but isn't the call to `systtemctl stop` of another unit a bit odd?
Perhaps I'm missing some advantages, but if this service needs to be started manually, couldn't a script be used for this same purpose?

As an alternative, what do you think of performing this cleanup conditionally on the stop of the `tezos-baking-custom` service?
I suppose that this could be done by adding a variable in `tezos-baking-custom.conf` (that one would set when filling up the `CUSTOM_NODE_CONFIG`) to choose whether to perform a cleanup and this variable could be looked up in an `ExecStopPost` script (to do the removal of the node data dir or not).
What do you think?

Reply:
> As an alternative, what do you think of performing this cleanup conditionally on the stop of the tezos-baking-custom service?

That sounds like a good option, I didn't think about it previously